search-website/README.md

```diff
@@ -0,0 +1,123 @@
+
+This README is an shortened version of the [full tutorial](https://aka.ms/search-website-tutorial). 
+
+Related resources: 
+* [Demo](https://victorious-beach-0ab88b51e.azurestaticapps.net/)
+* [Tutorial](https://aka.ms/search-website-tutorial)
+
+![Screenshot of sample web app](./images/web-app.png)
+
+You can easily deploy the sample onto Azure or run it locally by following the steps below.
+
+## Running the application locally
+
+To run the sample locally, follow the steps below.
+
+### Prerequisites
+
+- A GitHub account
+- [Node.js and Git](https://nodejs.org/)
+- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=shopathome-github-jopapa) installed
+- The [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions&WT.mc_id=shopathome-github-jopapa) installed
+- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=shopathome-github-jopapa) installed
+
+### Setup
+
+1. Clone (or Fork and Clone) this repository
+1. Open `./search-website` in Visual Studio Code. 
+1. Rename the `api/local.settings.json.rename` file to `api/local.settings.json`.
+
+    The `local.settings.json` file holds all of the keys that the application needs.
+    
+    For this sample, there is a search index using the [goodbooks-10k dataset](https://github.com/zygmuntz/goodbooks-10k) that that is publicly available using the connection information below. The index consists of 10,000 popular books that we'll search over in our application.
+    
+    ```json
+    {
+      "IsEncrypted": false,
+      "Values": {
+        "AzureWebJobsStorage": "",
+        "FUNCTIONS_WORKER_RUNTIME": "node",
+        "SearchApiKey": "REPLACE-WITH-YOUR-SEARCH-QUERY-KEY",
+        "SearchServiceName": "REPLACE-WITH-YOUR-SEARCH-RESOURCE-NAME",
+        "SearchIndexName": "good-books",
+        "SearchFacets": "authors*,language_code"
+      }
+    }
+    ```
+
+## Run the app locally
+
+This project can be run anywhere, but VS Code is required for local debugging.
+
+1. Open the `./search-website` application with VS Code.
+
+### Running the front-end
+
+1. From a terminal in the `./search-website` root, install front-end dependencies...
+
+   ```bash
+   npm install
+   ```
+
+1. Run the front-end project in the browser (automatically opens a browser window).
+
+   ```bash
+   npm start
+   ```
+
+### Running the API
+
+1. From a different terminal in the `./search-website/api` root, install front-end dependencies...
+
+   ```bash
+   npm install
+   ```
+
+1. Run the api project.
+
+   ```bash
+   npm start
```

Comment:
This failed for me: 
PS C:\search-website\api> npm install
npm WARN api@1.0.0 No repository field.
npm WARN api@1.0.0 No license field.

added 38 packages from 134 contributors and audited 38 packages in 1.746s
found 0 vulnerabilities

PS C:\search-website\api> npm start

> api@1.0.0 start C:\search-website\api
> func start

'func' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! api@1.0.0 start: `func start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the api@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\heidist\AppData\Roaming\npm-cache\_logs\2021-03-16T17_07_29_275Z-debug.log
PS C:\search-website\api>

Reply:
I'll update the dependencies to include Azure Function Core tools: 

```bash
npm install -g azure-functions-core-tools
```